trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Drag/parasite_drag_pylon.py

```diff
@@ -42,7 +50,8 @@ def parasite_drag_pylon(conditions,configuration,geometry):
 
     # Estimating pylon drag
     for propulsor in geometry.propulsors:
-        pylon_parasite_drag += pylon_factor *  conditions.aerodynamics.drag_breakdown.parasite[propulsor.tag].parasite_drag_coefficient
+        ref_area = propulsor.nacelle_diameter**2 / 4 * np.pi
+        pylon_parasite_drag += pylon_factor *  conditions.aerodynamics.drag_breakdown.parasite[propulsor.tag].parasite_drag_coefficient* (ref_area/geometry.reference_area * propulsor.number_of_engines)
```

Comment:
let's shorten these lines too, values like reference area and number of engines can be unpacked.  in the case of number of propulsors, unpack it in the loop.